docs/superpowers/plans/2026-05-23-mcpb-package.md

```
@@ -0,0 +1,367 @@
+
+The generated `dist/syslog-mcp-<version>-linux.mcpb` bundles the release
+`syslog` binary and launches it as:
+
+```bash
+server/syslog mcp
+```
+
+The bundle is query-only. It reads `syslog.db` from the configured data
+directory and does not start the syslog listener, HTTP MCP server, REST API,
+Docker Compose, or deployment flows.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)